frontend/src/metabase/querying/utils/drills/combine-columns-drill/components/ColumnAndSeparatorRow/ColumnAndSeparatorRow.module.css

```diff
@@ -0,0 +1,11 @@
+.separator {
```

Comment:
It's not clear to me which styling should go to CSS modules, and which should be overridden with `styles` prop in Mantine components. Suggestions are appreciated.